oci/copier.go

```diff
@@ -0,0 +1,114 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2024-Present Defense Unicorns
+
+
```

Comment:
```suggestion
```

We should run `make fmt` somewhere in our CI, could go in either `unit-tests` or `lint`.